collections/Bodl/MS_Bodl_851.xml

```diff
@@ -37,124 +37,780 @@
+                        Ramsey</persName>: ex libris, fol. 6v. It is not clear which portions of
+                     this composite manuscript the ex libris refers to, but if the owner is to be
+                     identified with the John Wells described by 'Dodsthorp' on fol. 2r, it 
+                     is reasonable to assume the whole of the present manuscript is referred to.</provenance>
+                  <provenance>Wells is identified by Macray, James (1914, p. xii), Rigg and others
+                     with John Wells (-1388), the opponent of Wycliffe (Emden III.2008; Emden did
+                     not accept the identification). This identification would allow at least some
+                     of the volume to be dated before 1388 (or before 1387, when Wells travelled to
+                     Italy), and perhaps localized to Oxford. Rigg suggested that the text of the
+                     bookplate on fol. 6v was written by the scribe of fols. 74ra-va, 81va,
+                     116ra-118rb, 118va-120vb, and 124r-139r, which does not seem to be correct, but
+                     the hand may be the same as the rubricator of codicological units I and II. It
+                     should be noted, however, that the John Wells who died in 1388 was buried at
+                     Perugia, whereas an addition on fol. 2r refers to (our?) John Wells as 'hic
+                     tumulatus', that is, presumably at Ramsey. If the rubric on fol. 94va refers to
+                     1384, that may provide a terminus a quo for that section of the manuscript; if the compiler of the Piers
+                     Plowman Z-text used the C-text, as suggested by S. Wood (see below) then the earliest
+                     date for unit III, and perhaps for other sections by the same scribe, may be 1388.
+                     Text and decoration suggest a date in the last quarter of the fourteenth
+                     century or perhaps the early fifteenth century.</provenance>
+                  <provenance><orgName role="fmo" key="org_121821230">Ramsey Abbey</orgName>:
+                     additions by Dodsthorp, partly relating to Ramsey Abbey, and presumably made at
+                     Ramsey: Dodsthorp is unidentified, but also signs an addition in another Ramsey
+                     book (MS. Bodl. 40, fol. 51v).</provenance>
+                  <provenance><q>Whyttynton</q>, 15th cent., fol. 6v in leadpoint.</provenance>
+                  <provenance><q>J. Kyngston</q>, 15th cent., fol. 208v in leadpoint, recorded by Rigg, now barely legible. The date 1505 (?) written in full, erased, fol. 208v.</provenance>
+                  <acquisition when="1601">Given by <persName role="dnr" key="person_3220">Cuthbert
+                        Ridley</persName> in 1601. Previous Bodleian shelfmarks: 'M. 5. 4. Art.',
+                     'Y. 1. 14. Th.', 'Arch. B. 95', 'Arch. B. 52', 'Arch. C. 29'.</acquisition>
```

Comment:
Should we be encoding these previous shelfmarks in a more structured way in the header?